README.adoc

```diff
@@ -158,32 +197,52 @@ userRegistry.xml
-
-It is not recommended to store passwords in plain text. The passwords in the [hotspot file=0]`userRegistry.xml` file are encoded using the Liberty `securityUtility` command with XOR encoding.
+The registry has four users, [hotspot=user-bob file=0]`bob`, [hotspot=user-alice file=0]`alice`, 
+[hotspot=user-carl file=0]`carl` and [hotspot=user-dave file=0]`dave`, and four groups, 
+[hotspot=group-name-Manager file=0]`Manager`, [hotspot=group-name-TeamLead file=0]`TeamLead`, 
+[hotspot=group-name-Employee file=0]`Employee` and [hotspot=group-name-PartTime file=0]`PartTime`. 
+Each user belongs to one or more groups.
 
-The [hotspot=application-bnd file=1]`server.xml` file contains the security configuration of the server.
+It is not recommended to store passwords in plain text. The passwords in the [hotspot file=0]`userRegistry.xml` file
+are encoded using the Liberty `securityUtility` command with XOR encoding.
 
 server.xml
 [source, xml, linenums, role='code_column']
 ----
 include::finish/src/main/liberty/config/server.xml[]
 ----
 
-Use the [hotspot=location file=1]`<include/>` element to add the basic user registry configuration to your server configuration. Open Liberty will include configuration information from the specified XML file in its server configuration.
+Use the [hotspot=location file=1]`<include/>` element to add the basic user registry configuration to your server configuration. 
+Open Liberty will include configuration information from the specified XML file in its server configuration.
 
-Map the groups in the user registry to the appropriate user roles supported by the application for proper user authorization. The [hotspot=group-name-Manager file=1]`Manager` and [hotspot=group-name-TeamLead file=1]`TeamLead` groups are mapped to the [hotspot=security-role-admin file=1]`admin` role while the [hotspot=group-name-Employee file=1]`Employee` group is mapped to the [hotspot=security-role-user file=1]`user` role using the [hotspot=Security file=1]`<security-role/>` and [hotspot=Group file=1]`<group/>` elements under the [hotspot=application-bnd file=1]`<application-bnd/>` configuration.
+The [hotspot file=1]`server.xml` file contains the security configuration of the server 
+under the [hotspot=application-bnd file=1]`<application-bnd/>` element.
+Map the groups in the [hotspot file=0]`userRegistry.xml` to the appropriate user roles supported by the application for proper user authorization,
+using the [hotspot=Security file=1]`<security-role/>` and 
+[hotspot=Group hotspot=group-name-Employee file=1]`<group/>` elements. 
```

Comment:
Change
"Map the groups in the [hotspot file=0]`userRegistry.xml` to the appropriate user roles supported by the application for proper user authorization, using the [hotspot=Security file=1]`<security-role/>` and [hotspot=Group hotspot=group-name-Employee file=1]`<group/>` elements. 
To
"Use the [hotspot=Security file=1]`<security-role/>` and [hotspot=Group hotspot=group-name-Employee file=1]`<group/>` elements to map the groups in the [hotspot file=0]`userRegistry.xml` file to the appropriate user roles supported by the application for proper user authorization. "